src/adapters/github.ts

```diff
@@ -309,7 +309,10 @@ class GithubAdapter extends GitAdapter {
   }
 
   protected getRepositoryUrl(repo: IRepo): string {
-    return `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`;
+    return (
+        `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`
```

Comment:
As this variable can hold either the GitHub public, Enterprise Cloud, or Enterprise Server URL, I'm thinking a more semantic variable name would be `githubUrl`.